CMakeLists.txt

```
@@ -14,11 +14,8 @@
 #######
 
 # Set minimum Cmake version and setup policy behavior
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.10)
```

```suggestion
cmake_minimum_required(VERSION 3.10...4.0)
```
If you wish to reduce maintenance. It does help test that the project is compliant with all the policies up to the higher-bound and it makes downstream packaging much easier.

But a 3.10 is actually a good minimum as well because that covers the next round of policy deprecation.